DoctorWho.Db/Repositories/AuthorsRepository.cs

```diff
@@ -0,0 +1,30 @@
+﻿using DoctorWho.Domain;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DoctorWho.Db.Repositories
+{
+    public class AuthorsRepository
+    {
+        public static void CreateAuthor(string authorName)
+        {
```

Comment:
we usually create instances of the Dbcontext whenever we need to use it, and it should be in the class level, not it's class level, let's discuss that idea in the meeting